linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml

```
@@ -29,6 +29,7 @@ identifiers:
 references:
     nerc-cip: CIP-003-3 R4.2,CIP-007-3 R5.1
     nist: SC-13,SC-12(2),SC-12(3)
+    stigid@rhel8: RHEL-08-010200
```

```suggestion
    stigid@rhel8: RHEL-08-010020
```

I think you meant `RHEL-08-010020` for all these items